CK3toEU4/Data_Files/configurables/culture_map.txt

```diff
@@ -7,16 +7,15 @@
 
 # North Germanic
 
+link = { eu4 = icelandic ck3 = norse ck3 = norwegian ck3 = danish ck3 = swedish province = 370 province = 371 tech = western gfx = westerngfx }
 link = { eu4 = swedish ck3 = swedish tech = western gfx = westerngfx }
 link = { eu4 = norwegian ck3 = norwegian tech = western gfx = westerngfx }
 link = { eu4 = danish ck3 = danish tech = western gfx = westerngfx }
-link = { eu4 = icelandic ck3 = norse ck3 = norwegian ck3 = danish ck3 = swedish province = 370 province = 371 tech = western gfx = westerngfx }
 link = { eu4 = hannoverian ck3 = old_saxon tech = western gfx = westerngfx }
 link = { eu4 = norse ck3 = norse tech = western gfx = westerngfx }
 
 # German: the split
-link = { eu4 = swabian ck3 = german ck3 = saxon ck3 = franconian ck3 = bavarian ck3 = swabian tech = western gfx = westerngfx province = 68 province = 4765 province = 4714 province = 4713 province = 4711 province = 1880 province = 1872 province = 1868 }
```

Comment:
This reordering was because Alsatian has areas that overlap with Vanilla EU4 Swabian

Reply:
So you want the priority to be for the Swabian then? Very well.

Reply:
Priority is for Swabian because Swabian is in vanilla, Alsatian isn't. Alsatian has been added for German leaking into Lorraine/Champagne.